examples/run.py

```diff
@@ -0,0 +1,90 @@
+# Copyright (c) 2020 Fredy Ramirez <https://formateli.com>
+# For license see LICENSE
+"""
+run.py
+Show all examples located in this example folder.
+Main window show a button for each example.
+"""
+import sys, os
```

Comment:
Please put each import of a module on a newline for [PEP-8 compliance](https://www.python.org/dev/peps/pep-0008/#imports).